apps/src/lab2/lab2Redux.ts

```diff
@@ -227,10 +226,10 @@ const labSlice = createSlice({
       state.channel = action.payload;
     },
     setHideShareAndRemix(state, action: PayloadAction<boolean>) {
```

Comment:
We should also remove these setters (setHideAndShareRemix, setIsProjectLevel) and just set levelProperties directly. We can actually probably just do that in `onLevelChange` - instead of passing in `levelData` and `appName`, we can just pass in `levelProperties` and set it on the state object.